spotify_player/src/event/page.rs

```diff
@@ -501,15 +501,15 @@ pub fn handle_navigation_command(
 
     let configs = config::get_config();
     match command {
-        Command::SelectNextOrScrollDown => {
-            if id + 1 < len {
-                page.select(id + 1);
+        Command::SelectNextOrScrollDown { count } => {
+            if id + count < len {
+                page.select(id + count);
```

Comment:
should this be `page.select(min(id + count, len - 1))`? I assume that will be more user friendly